fhttp/http_test.go

```diff
@@ -872,7 +917,29 @@ func TestDefaultHeadersAndOptionsInit(t *testing.T) {
 	}
 }
 
-// --- for bench mark/comparaison
+func TestChangeMaxPayloadSize(t *testing.T) {
```

Comment:
that part (first commit) is just a move of that test to the right section 
see https://github.com/istio/fortio/pull/219/commits/b22958f8c3ab0a8a87fef075ebb0d5f7bf7dfff8 for the interesting part